src/libutil/unix/environment-variables.cc

```diff
@@ -1,4 +1,4 @@
-#include <cstdlib>
+#include <stdlib.h>
```

Comment:
Why this? `<cstdlib>` is standard right?

Reply:
Oops, just bad conflict resoution